{% extends "layout.html" %}
{% block subtitle %} Lesson 5 {% endblock %}

{% block includes %} 

<script>

$(document).ready(function(){
    $("#mutbutton").click(function(){
        $("#mutDiv").toggle();
    });
    $("#selbutton").click(function(){
        $("#selDiv").toggle();
    });
    $("#GDbutton").click(function(){
        $("#GDDiv").toggle();
    });
});


</script>

{% endblock %}

{% block body %}

<h1> Genetic Drift, Mutation, Selection</h1>

<h3> Genetic Drift in a real population </h3>

Let us now examine the distribution of allele frequencies across multiple generations in a real population. It turns out that the equations introduced for modelling particle diffusion can be used to model this process and recapitulate the observed genetic drift. Let us first take a look at the experimental data. These data are obtained from table 3 in <i>Buri P., Evolution, Vol. 10, No. 4 (Dec., 1956), pp. 367-402.</i>. We <a href="{{ url_for('static', filename='buri_data_summary.html') }}">extracted and plotted </a> them in MATLAB for you. Note the similarity between evolution of the allele frequency distributions to our particle positon distributions from <a href="/lesson1">lesson 1 </a> and <a href="/lesson2">lesson 2 </a>. <br>

<a href="http://www.rpgroup.caltech.edu/publications/Garcia2016.pdf"> Garcia et al.</a> did a great job summarizing Buri's experiment, which are described in the figures below. Buri started his experiment with 107 populations of 8 male and 8 female flies that are heterozygous in a gene that determines eyecolor. Accordingly there are 16 heterozygotes with 32 alleles at the diploid bw locus in each population. If the individuals are homozygous in the bw allele they have white eyes, bw75 homozygotes have red eyes and heterozygotes have organge eyes.

 <figure style='float: left; padding: 10px 10px 10px 10px; width: 60%'>
  <img style='float: left' src="{{ url_for('static', filename='buri_scheme.png') }}" width="43%" >
  <img style='float: left' src="{{ url_for('static', filename='buri_genes.png') }}" width="57%" >
    <img src="{{ url_for('static', filename='buri_data2.png') }}" width="100%" >    
  <figcaption><p style="font-size: 6pt"></p></figcaption>
</figure>
<br>
Buri allowed the flies to mate and lay eggs and then removed the parent generation and let larvae hatch. He randomly selected 8 males and 8 females and transferred them in new vials and counted the number of homozygotes and heterozygotes by examining eye color. As discussed in lesson 4, you can compute the allele frequencies from genotype frequencies.
He charted the allele frequencies for all 107 populations over 20 generations. Buri found that in most vials the eye color is fixed, after 20 generations all individuals were homozygous. This observations is what Fisher and Wright described as Genetic Drift. Click 
<a href="http://highered.mheducation.com/sites/dl/free/0072835125/126997/animation45.html">here</a> for an animation that explains Genetic Drift. For the interested reader <a href="http://www.nature.com/scitable/topicpage/genetic-drift-and-effective-population-size-772523">Nature's Scitable</a> is a great start for further studies. Fixation of alleles is analogous to particles sticking to a boundary once they diffused into this boundary. Using this boundary condition, the equations we introduced in <a href="/Lesson3">Lesson 3</a> can be used to model genetic drift.  <a href=" http://malooflab.phytonetworks.org/apps/genetic-drift-markov/"  target="_blank">Here</a>  you can  interactively plot the theoretical allele frequencies for various conditions. Can you modify the parameters and recapitulate Buri's data?

<br><br><br><br><br><br><br><br><br><br><br><br><hr>
<h3> Selection in a real population </h3>

In another experiment Cavener et al chose a very similar experimental approach. They however run the experiment twice, in the absence or prensence of ethanol and monitored the frequency of two different Alcohol Dehydrogenase alleles. 
By including the ethanol the fitness of individuals with the less efficient Alcohol Dehydrogenase is decreased (see inset below - Figure 16 of 
<a href="http://www.rpgroup.caltech.edu/publications/Garcia2016.pdf"> Garcia et al.</a>).  You can explore the effect of small population size, fitness and mutation rate <a href="https://evobir.shinyapps.io/wf_model/">here</a>. Again, note the similarity to <a href="https://evobir.shinyapps.io/brownian_motion/">brownian motion </a>.


<center>
<figure style='align: center ; margin: 0 30px 50px 0; width: 80%'>
<img src="{{ url_for('static', filename='buri_selection.png') }}" width="100%" >
  <figcaption><p style="font-size: 6pt"></p></figcaption>
</figure>
</center>


<hr>
<h3> Genetic Drift, Selection, and Mutation</h3>
<p>
The three major factors that influence the evolution of a population are: genetic drift, selection, and mutation. In this section, we will explore the effects of these factors on terms in the drift-diffusion equation.
</p>

<p>
<abbr class="wikihover" rel="Genetic_drift">Genetic Drift</abbr> is unfortunately named, since it has nothing to do with drift in the physical sense as discussed for the past coupled lessons. As we saw above, genetic drift happens due to <em>random fluctuations</em> in which members of a population are able to reproduce, as lucky or unlucky individuals have many or few surviving children. We saw above that drift introduces pure diffusion terms into the allele frequency dynamics.
</p>

<p>
<abbr class="wikihover" rel="Natural_selection">Selection</abbr> is the what most people think of when they think of evolution. As we saw above, selection is the result of competition among members of the population (for example, for a finite amount of resources), which enriches for the <abbr class="wikihover" rel="Fitness_(biology)">fittest</abbr> members of the population. Here, fitness is a function of the phenotype of the organism, that is to say, whatever feature of the organism interacts with the environment. We saw above that selection introduces pure drift terms into the allele frequency dynamics.
</p>


<p> Finally, there is mutation. <abbr class="wikihover" rel="mutation">Mutation</abbr> is the introduction of random errors into an organism's genome. Because they are random, many of the errors may not have an effect on the exact phenotype that we are interested in. The effective mutation rate can be modeled as the base rate times the probability of getting a mutation in genes that we are interested in, but, in general, mapping between <abbr class="wikihover" rel="Genotype-phenotype distinction"> genotype and phenotype</abbr> is <em> very </em> difficult to model. Mutation becomes important on longer timescales than what was studied above. Mutation introduces both a drift and a diffusion term into the allele frequency dynamics. The diffusion is due to the continuous introduction of random allele changes, while the drift term is a reflection of the fact that mutation pushes the allele frequency towards a balance. This balance occurs when both alleles are introduced at the same rate by mutation. So, the mutational drift term is a function of the current state of the population.
</p> 

<br>
<br>
<br>
<br>
<br>
{%endblock %}